INSTALL.md

```diff
@@ -104,7 +104,7 @@ For bare-metal installations, the full instructions for setting up PostgreSQL, T
     sudo apt-get install -y git
     git clone https://github.com/timescale/promscale_extension
     cd promscale_extension
-    git checkout 0.7.0
+    git checkout 0.7.1-dev
```

Comment:
why are we changing the instructions to install a non-release? Also, I don't think thats an actual tag.

Reply:
You are correct. `./update-version.sh` does it, but it needs to leave `INSTALL.md` alone on the post-release run. I'll fix it.